other/n_body_simulation.py

```diff
@@ -0,0 +1,262 @@
+"""
+In physics and astronomy, a gravitational N-body simulation is a simulation of a

```

Comment:
Should we have `physics` and/or (even better) `astronomy` directory so that we do not grow the `other` directory.

Reply:
If we make it `physics`, we could also include the files from `electronics` and remove one root-directory. But in any case, I think `physics` is the better choice since this is not specifically about celestial objects.

Reply:
Let's add `physics` with this algorithm but let's ___not___ migrate `electronics` into it.